xml/art_raspberry-pi.xml

```diff
@@ -743,49 +644,25 @@ Password:
     </step>
     <step>
      <para>
-      Finally, you are offered the option to configure a wireless network.
+      Finally, you have the option to configure a wireless network.
       If you have a wired Ethernet connection, you can skip this step by selecting <guimenu>No</guimenu>.
      </para>
      <informalfigure>
       <mediaobject>
        <imageobject>
         <imagedata width="80%" fileref="rpi_first_wifi.png"/>
        </imageobject>
-       <textobject><phrase>&jeos-firstboot; asking you whether to configure a wireless network</phrase>
+       <textobject><phrase>&jeos-firstboot; prompting to configure a wireless network</phrase>
        </textobject>
       </mediaobject>
      </informalfigure>
      <para>
-      If you select <guimenu>Yes</guimenu>, you will see the following screens:
+      To configure a wireless network, press <guimenu>Yes</guimenu>.
      </para>
-     <remark>
-      cwickert 2020-01-22: First step is commented out because rpi_first_wifi_interface.png
-      needs an update (see https://bugzilla.suse.com/show_bug.cgi?id=1156663 and I'm not
-      sure the step is still necessary. Why select a device if there is only one?
-     </remark>
      <substeps>
-      <!--
-      <step>
-       <para>
-        Choose the network interface <guimenu>wlan0</guimenu> by selecting <guimenu>OK</guimenu>.
-       </para>
-       <informalfigure>
-        <mediaobject>
-         <imageobject>
-          <imagedata width="80%" fileref="rpi_first_wifi_interface.png"/>
-         </imageobject>
-         <textobject>
-          <phrase>
-           &jeos-firstboot; asking you to select a wireless network device
-          </phrase>
-         </textobject>
-        </mediaobject>
-       </informalfigure>
-      </step>
-      -->
       <step>
        <para>
-        Select a wireless network to connect to and proceed with
+        Select the desired network interface and proceed with
```

Comment:
You're changing the meaning here: You deleted the section about selecting a network interface above - this would then still be about selecting the SSID, no?

Reply:
Good point! I'll revert to the original wording.